libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1974,6 +1986,17 @@ MAV_RESULT GCS_MAVLINK::handle_command_mag_cal(const mavlink_command_long_t &pac
     return compass->handle_mag_cal_command(packet);
 }
 
+MAV_RESULT GCS_MAVLINK::handle_command_request_autopilot_capabilities(const mavlink_command_long_t &packet)
+{
+    const AP_FWVersion &fwver = get_fwver();
```

Comment:
**Huge nitpicking**: do this after the *if*

Reply:
Done.